lib/telephony/alert_sender.rb

```diff
@@ -1,5 +1,5 @@
 module Telephony
-  class AlertSender
+  class AlertSender < BasicSender
```

Comment:
Thanks for updating based on the feedback. I feel like we could accomplish the same goals in this PR if we exposed `send_raw_message` on `AlertSender` and planned to use that? that way we wouldn't need to create additional classes and create that indirection?

Reply:
@zachmargolis, ok, we can.

Given what I saw the AlertSender, OptSender can be a subclass of what BasicSender for more specific invocations here if possible.

But if people think subclassing is unfavorable, I am fine with it.

Reply:
@zachmargolis , removed and added one single method.

Reply:
Most other methods in AlertSender can call send_raw_message.